### ISSUE TITLE
Version v3.1.4 is missing

### ISSUE THREAD

**Issue by goran-mahovlic** [kost/fujprog#13]
That is newest version produced...

**Comment by dpavlin** [comment]
c7268a2bced5639c99647a27d175d4cb818952c7 with different version should do the trick

**Comment by goran-mahovlic** [comment]
Yes, but maybe we could figure out some other way, as with each version we have bigger and bigger core, and actually everything on HW is still the same :)

**Comment by dpavlin** [comment]
We asked @emard about that and we got confirmation that chamges will increase minor version so we could have just v3.0.* and v.3.1.*.

I had idea to implement ? as wildcard so we could define both fpga size as ??K and version as v3.?.?. However, code doesn't list all devices and then select one, but tries all cable_path(s) defined in source it's not as simple as replacing strcmp with custom wildcard version.

**Comment by emard** [comment]
If you can get any "*" solution that would be great
New releases don't come so often, but more I forget
to update sources/binarys
